sample/simple-videoplayer-channel/README.md

```diff
@@ -0,0 +1,17 @@
+# Roku Sample Channel
+
+This Roku channel project demonstrates the usage of the Adobe Roku SDK.
+
+To run the project on your Roku device, follow the steps below to set it up beforehand.
+
+## Install the Roku SDK
+
+To install the Roku SDK in this sample project, you can copy the SDK files to this sample project by following the [install instruction](../../Documentation/getting-started.md).
+
+## Initialize and configure the Roku SDK
+
+The Roku SDK is initialized and configured in the [SimpleVideoScene](./components/SimpleVideoScene.brs) file. The default `configId` value is invalid. Please provide a valid value for it.
```

Comment:
`The Roku SDK is initialized in the [SimpleVideoScene](./components/SimpleVideoScene.brs) file. Configuration is empty, update the configuration with valid configId before running the channel.`
~The default `configId` value is invalid. Please provide a valid value for it.~